source/core/slang-type-text-util.cpp

```diff
@@ -160,6 +191,54 @@ static const ScalarTypeInfo s_scalarTypeInfo[] =
     return UnownedStringSlice::fromLiteral("unknown");
 }
 
+/* static */SlangCompileTarget TypeTextUtil::asCompileTargetFromExtension(const UnownedStringSlice& slice)
```

Comment:
Just my preference, but I'd prefer to call this something like `findCompileTargetForExtension` or `findCompileTargetFromExtension`, where "find" both implies that there is some searching going on, as well as that the search might not result in anything being found. Using `as` here implies a cast to me, which isn't quite what this is.

Reply:
Fair point.